Sources/xcproj/XCScheme.swift

```diff
@@ -169,6 +169,56 @@ final public class XCScheme {
         }
     }
 
+    final public class EnvironmentVariables {
```

Comment:
Not sure if we need this class, as it only simply holds an array of `[EnvironmentVariable]`.

Maybe we can factor this and `CommandLineArguments` away later